index.js

```diff
@@ -16,10 +16,16 @@ const GoogleLogs = require('./logs/googleLogs');
 const GoogleInfo = require('./info/googleInfo');
 
 class GoogleIndex {
-  constructor(serverless, options) {
+  constructor(serverless, options, v3Utils) {
     this.serverless = serverless;
     this.options = options;
 
+    if (v3Utils) {
```

Comment:
This is setup on `GoogleIndex` level where everywhere in the code you reference it from `this.provider` - I believe it's not reachable on `GoogleProvider` level.